README.md

```diff
@@ -1,7 +1,9 @@
-# Go Databricks SQL Driver
+# Preview Go Databricks SQL Driver
 
 Databricks SQL driver for Go's [database/sql](https://golang.org/pkg/database/sql) package.
 
+**NOTE: This Driver is in Preview mode.**
```

Comment:
I'm not sure "Preview Mode" will be clear to people? Even if they are familiar with the [Databricks definitions](https://docs.databricks.com/release-notes/release-types.html) - we don't really map to any of the preview phases...

Maybe we should call it beta?

Something like: "Note: this driver is in beta."?

Reply:
changed to beta.